demo/app/main-page.ts

```diff
@@ -1,32 +1,32 @@
-import { FlashLight } from '@essent/nativescript-flashlight';
-import { EventData, Observable, PropertyChangeData } from 'data/observable';
-import { Page } from 'ui/page';
-import { Slider } from 'ui/slider';
+import { Flashlight } from 'nativescript-flashlight';
```

Comment:
This should be @essent/nativescript-flashlight

Reply:
It will not recognize the module if I add @essent in front of it